bentoml/yatai/client/bento_repository_api.py

```diff
@@ -273,12 +346,34 @@ def _update_bento_upload_progress(
         self.yatai_service.UpdateBento(update_bento_req)
 
     def download_to_directory(self, bento_pb, target_dir):
-        if bento_pb.uri.s3_presigned_url:
-            bento_service_bundle_path = bento_pb.uri.s3_presigned_url
-        elif bento_pb.uri.gcs_presigned_url:
-            bento_service_bundle_path = bento_pb.uri.gcs_presigned_url
+        """
+        Download or move bundle bundle to target directory
+
+        Args:
+            bento_pb: bento bundle protobuf dict
+            target_dir: path
+
+        Returns:
+            None
+        """
+        if bento_pb.uri.type == BentoUri.S3 or bento_pb.uri.type == BentoUri.GCS:
+            bento_service_bundle_path = (
+                bento_pb.uri.s3_presigned_url
+                if bento_pb.uri.type == BentoUri.S3
+                else bento_pb.uri.gcs_presigned_url
+            )
+        elif bento_pb.uri.type == BentoUri.LOCAL:
```

Comment:
Can we rename `BentoUri.LOCAL` to `BentoUri.FILE_SYSTEM` in this PR?

Reply:
This change might impact a lot of files. Let's do it in a separate pr.